streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/common/AbstractID.java

```diff
@@ -0,0 +1,123 @@
+package org.ray.streaming.runtime.core.common;
+
+import java.io.Serializable;
+import java.util.Random;
+
+/**
+ * Streaming system unique identity base class.
+ * For example, ${@link org.ray.streaming.runtime.core.resource.ContainerID }
+ */
+public class AbstractID implements Comparable<AbstractID>, Serializable {
+
+  private static final long serialVersionUID = 1L;
+  private static final Random RANDOM = new Random();
+  private static final int SIZE_OF_LONG = 8;
+  private static final int SIZE_OF_UPPER_PART = 8;
```

Comment:
Can you explain why `id` consist of two 8-byte long value? Can't we express it as a single int or long value?

Reply:
This is design standard. We can choose 8 or 16 bits as the unique id.